README.md

```diff
@@ -1 +1,153 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+>A linter is a tool that checks your code for errors and provides suggestions for how to improve your code. By using a linter, you can identify and fix issues before they become larger problems.
+
+You can configure a linter to check multiple files in your project, which is especially helpful when working on larger projects with many files. You can also use the configuration file of your linter as a blueprint for your project's coding conventions finally improving the overall quality of your code and making your development process more efficient.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1.Github account 
+2.Text Editor 
+3.Web Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Usage
+
+To run the project, execute the following file: index.html
+
+### Run tests
+
+To run tests, run the following command:
+To run tests, run the following command: Open the code in Visual Studio Code and run the following commands :
+
+npx hint . npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using: github, VS Code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="Vivek Raj"></a>
+
+👤 **Vivek Raj**
+
+- GitHub: [@githubhandle](https://github.com/VivekRaaZ?tab=repositories)
+- Twitter: [@twitterhandle](https://twitter.com/BossVivekRaj)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/vivek-raj-a78b92256/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improving the outlook by implementing the responsive design]**
+- [ ] **[Styling the heading by using CSS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> The beginning is always difficult, and once the flow begins, then no one can stop it. So do support this first step towards excellence.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thanks microverse team, github team and my coding partners, who contributed towards the completion of this task.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is copyright of [Vivek Raj]
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
[OPTIONAL] Hey 👋🏽 this note is only to give you directives so you can remove it ❌ once you're done reading the directives as it becomes of no use anymore.